### ISSUE TITLE
Thinking of removing the overlay

### ISSUE THREAD

**Issue by Moerill** [Moerill/token-mold#71]
I am thinking of removing the token mold overlay from the module. 
Why?
There exists this [awesome tooltip module](https://github.com/bmarian/token-tooltip-alt), doing almost all thing that i want from my overlay, with more options (more incoming with v2.0 "soon"!) to customize, most issues existing here resolved and already having a few features that i was thinking of adding. 
This would also fit to my "more modern" approach of thinking about module development - more feature focused modules, not big feature suites, like Token Mold was.

I'm also talking with the creator of adding a "overlay mode", where the tooltip is displayed above the token instead of on the sides.

TLDR: please check out [Token Tooltip Alt](https://github.com/bmarian/token-tooltip-alt) as alternative to token molds overlay and leave your thoughts here, about what is missing or why you'd like/dislike the switch.

**Comment by SalieriC** [comment]
I especially like Token mold for its clean looking overlay which fits modern and medieval games alike. I originally installed Token Mold specifically for this feature, everything else was pretty much irrelevant for me and still is. I really like the transparent overlay, much better than the module you mentioned imo. Even if that one gets an overlay feature the background would still annoy me (I don't know if this is configurable as I've never used it). If the overlay gets removed from Token Mold I don't see any more reason to use it at all.

**Comment by Moerill** [comment]
> I really like the transparent overlay, much better than the module you mentioned imo.

The dark theme is kinda transparent, and already close in look to token molds overlay. 

>  If the overlay gets removed from Token Mold I don't see any more reason to use it at all.

which is fine, no reason to use a big module with many features you don#t use, if a one feature module does the job (better).

**Comment by apoapostolov** [comment]
I really love ho far  Token Tooltip Alt has gone, but the external panel has always struck me as busy and i much loved the in-token way to show statistics, fewer but the most important to each game. I will be a bit sad if  Token Tooltip Alt cannot implement the exact same way of visualization, and I think they might not since they are so focused on the external panel. This is why I am a bit sad to see that functionality retired.

**Comment by Moerill** [comment]
> This is why I am a bit sad to see that functionality retired.

the title is "**Thinking** of removing the overlay". its not set in stone Also its a slow progress, but i do feel that investing in that module to be awesome and do the job *could* be the better way instead of trying to fix some of the issues i currently have.

>  i much loved the in-token way to show statistics

i am talking with Marian, and trying to push him to add those things. As for the overlay instead of tooltip, i'm going to probably do that on my own and send it to him to include.

**Comment by Moerill** [comment]
But thanks to all for commenting on this - i'm happy to hear that my current approach is liked that much!

**Comment by apoapostolov** [comment]
If Marian adds in-token visualization the way Token Mold does it, I would happily migrate. I love how far Token Tooltip Alt has gone in power user customization.

**Comment by apoapostolov** [comment]
Hello,
Based on this response https://mail.google.com/mail/u/0/#inbox/FMfcgxwKjBNlZkZWGjpZDGsTMmnMJpTg I do not think Marian understood your proposal, or is willing to have completely different implementation to match yours. I would suggest keeping it along as long as it works. Thanks in advance.

**Comment by Moerill** [comment]
While i can't follow your mail link, buuuut after a lengthy discussion he will still add it. It won't be exactly the same, since it will be fixed on the left of the token and not on the right, but i do think thats acceptable :sweat_smile:

**Comment by SalieriC** [comment]
> I can't follow your mail link, buuuut after a lengthy discussion he will still add it. It won't be exactly the same, since it will be fixed on the left of the token and not on the right, but i do think thats acceptable

From what I see in the preview of his current update ( https://discordapp.com/channels/170995199584108546/648215359895240715/773529451480219698 ) it is nowhere near as sleek and elegant as the one from Token Mold. I would not want to use that as it still covers the surrounding area. :-S

**Comment by Moerill** [comment]
![image](https://user-images.githubusercontent.com/26152015/98117372-ef6aeb00-1ea9-11eb-8acc-688a8adde656.png)

i do think it can be pretty close, besides positioning. (even a bit nicer looking imo!) For my taste it mainly needs to get rid
The main difference is, that the tooltip keeps it size independent from zoom level. (which is actually a thing i wanted to do for token mold as well, since i find it very annoying to not be able to read a tooltip on a quick glance when zoomed out, which i'm most of the time.)
Here an example zoomed in completly, with both the tooltip and the token mold overlay active:
![image](https://user-images.githubusercontent.com/26152015/98116814-1c6ace00-1ea9-11eb-99db-385f7b7eee32.png)
And here zoomed out, to a zoom lvl which i'm often working with, for the purpose of being able to see both i moved the tooltip to be besides the token
![image](https://user-images.githubusercontent.com/26152015/98116924-4c19d600-1ea9-11eb-95fb-a0d2a658713a.png)

Its not completly there *yet*, there are still a few small changes i'd like to be included, like a better selection menu for icons and even spacing for the icons used

**Comment by SalieriC** [comment]
> What is what you're still missing/thinking is sleeker for the token mold overlay?

From the video I linked it seems that the overlay still covers the surrounding area when there is more information in it than fits inside of one square. I admit that reading your overlay when zoomed out can be a bit hard but I rather have that than covering the surrounding area.

**Comment by Moerill** [comment]
Token Mold is doing the same, if you have to much information it will spill over:
![image](https://user-images.githubusercontent.com/26152015/98118937-378b0d00-1eac-11eb-8b7a-c0b6d19a7707.png)

But ye it seems the relative size version *will* come at some point in the future.

For the time being i'll keep token molds overlay, but will probably not put much work into updating it, while i observe how token tooltip develops.

**Comment by jalensailin** [comment]
One thing i like about your module is the drop-down menu for choosing which attributes are displayed. I understand that token tooltip alt allows you to type it in yourself because it allows for more complex expressions but I find that having to manually type in attributes is just annoying and thats honestly why I opt for Token Mold's overlay. That and its simplicity

**Comment by ThingsCouldGetDicey** [comment]
well the thing that was missing from the overlay is to distinguish between npcs/pcs so that you could have more information on the npcs and monsters than you need on the pcs.
